features/testbot/nil.feature

```diff
@@ -0,0 +1,42 @@
+@routing @testbot @nil
+Feature: Testbot - Check assigning nil values
+    Scenario: Assign nil values to all way strings
+        Given the profile file "testbot" extended with
+        """
+        function way_function (way, result)
```

Comment:
this looks like we could easily create strange interaction with `way_function` also defined in the `testbot` profile. Is this guaranteed to replace the previously defined `way_function` or should we do a full dedicated profile here?